rust/src/lib/route.rs

```diff
@@ -104,6 +105,51 @@ impl Routes {
         }
         Ok(())
     }
+
+    pub(crate) fn resolve_next_hop_iface_ref(
+        &mut self,
+        merged_ifaces: &MergedInterfaces,
+    ) -> Result<(), NmstateError> {
+        let iface_name_search = InterfaceNameSearch::new(merged_ifaces);
+
+        if let Some(config_routes) = self.config.as_mut() {
+            for route in config_routes.iter_mut().filter(|r| !r.is_absent()) {
+                let route_display_str = route.to_string();
+                if let Some(nex_hop_iface) = route.next_hop_iface.as_mut() {
```

Comment:
How about changing the variable to `next_hop_iface` to improve clarity?

Reply:
Fixed.